tests/framework/extensions/provisioning/ssh.go

```diff
@@ -81,6 +95,27 @@ func CallSSHTestByName(testCase provisioninginput.SSHTestCase, node *nodes.Node,
 		}
 
 		return err
+
+	//This test checks if the audit log file is properly created on the node (skipped if its not a control plane node).
+	case auditLog:
+		if node.NodeLabels[controlPlaneLabel] != "true" {
+			logrus.Infof("Node %s is not a control-plane node skipping", node.PublicIPAddress)
+			return nil
+		}
+
+		logrus.Infof("Running audit log test on node %s", node.PublicIPAddress)
+		output, err := node.ExecuteCommand(checkAuditLogCommand)
+		if err != nil && !errors.Is(err, &ssh.ExitMissingError{}) {
+			return errors.New(err.Error() + output)
+		}
+		strOutput := output[:strings.IndexByte(output, '\n')]
+		if strings.TrimSpace(strOutput) != auditLogDir {
+			return errors.New("no audit log file found")
+		}
+		logrus.Infof("Audit log file: %v", strOutput)
```

Comment:
Can the logging message be changed to something like `Successfully found audit log file %s on node %s`?